tests/test_pe.py

```diff
@@ -62,6 +62,7 @@ def write(self, string):
         self.output[key] = value
         return len(string)
 
+IS_FAST_TEST = 'QL_FAST_TEST' in os.environ
```

Comment:
I think we can remove the QL_FAST_TEST

forgotten why its there and never use it ever since.

Reply:
skip malware in CI ? https://github.com/qilingframework/qiling/pull/372

Reply:
This is quite helpful for local testing, which I always run before pushing any changes.
I found about it just recently, and noticed it is broken (i.e. defining that environment variable would cause the test to return `None` instead of `True` and fail the assertion).